### PR TITLE
change title on README

### PR DESCRIPTION
**Pull request by AdemolaBTheG**

### CODE REVIEW

**Comment by Flingex** [comment]
WOW

**Comment by Flingex** [comment]
O

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## Welcome to GitHub Games
+## Welcome to awesome game of Tetris
 
 A project based learning activity for people who are getting started with Git and GitHub.
```
Comment (Flingex) [inline README.md:3]: Lets go

**Review by AdemolaBTheG — COMMENTED** [review]
Suadadakdoa

**Review by AdemolaBTheG — COMMENTED** [review]
sac<<x<x<x<x<x<<

**Review by AdemolaBTheG — COMMENTED** [review]
3q4qr

**Pull request merged**